docs/man/monerod.1

```diff
@@ -0,0 +1,5343 @@
+.TP
+\fB\--detach\fR
+Run as daemon
+.TP
+\fB\--non-interactive\fR                     
+Run non-interactive, Do not require tty in a foreground mode. Helpful when running in a container. By default \fBmonerod\fR runs in a foreground and opens stdin for reading.
+.TP
+\fB\--no-zmq\fR
+Disable ZMQ RPC server
+.TP
+\fB\--no-igd\fR                    
+Disable UPnP port mapping
+.TP
+\fB\--max-txpool-weight\fR arg (=648000000)
+Set maximum txpool weight in bytes. These are transactions pending for confirmations (not included in any block).
+.TP
+\fB\--enforce-dns-checkpointing\fR
+checkpoints from DNS server will be enforced, The emergency checkpoints set by \fBMoneroPulse\fR operators will be enforced. It is probably a good idea to set enforcing for unattended nodes.
+If encountered block hash does not match corresponding checkpoint, the local blockchain will be rolled back a few blocks, effectively blocking following what MoneroPulse operators consider invalid fork. The log entry will be produced: \fBERROR Local blockchain failed to pass a checkpoint, rolling back!\fR Eventually, the alternative (“fixed”) fork will get heavier and the node will follow it, leaving the “invalid” fork behind.
+.TP
+\fB\--disable-dns-checkpoints\fR
+Do not retrieve checkpoints from DNS
+
+
+.SS Logging:
+.TP
+\fB\--log-file\fR arg
+.br
+(=~/.bitmonero/bitmonero.log,
+.br
+~/.bitmonero/testnet/bitmonero.log if 'testnet',
+.br
+~/.bitmonero/stagenet/bitmonero.log if 'stagenet')
+.br
+Specify Full path log file
+.TP
+\fB\--log-level\fR arg (=0)
+0-4 with 0 being minimal logging and 4 being full tracing. These are general presets and do not directly map to severity levels.
+.TP
+\fB\--max-log-file-size\fR arg (=104850000)
+Specify maximum log file size [B], Once log file grows past that limit, \fBmonerod\fR creates the next log file with a UTC timestamp postfix -YYYY-MM-DD-HH-MM-SS.
+.TP
+\fB\--max-log-files\fR arg (=50)
+Specify maximum number of rotated log
+files to be saved (no limit by setting to 0)
+
+
+.SS monerod Performance:
+.TP
+\fB\--sync-pruned-blocks\fR
+Allow syncing from nodes with only pruned blocks
+.TP
+\fB\--prune-blockchain\fR
+Prune blockchain
+.TP
+\fB\--db-sync-mode\fR arg (=fast:async:250000000bytes)
+Specify sync option, using format [safe|fast|fastest]:[sync|async]:[<nblocks_per_sync>[blocks]|<nbytes_per_sync> [bytes]].
+.TP
+\fB\--max-concurrency\fR arg (=0)
+Max number of threads to use for a parallel job
+.TP
+\fB\--prep-blocks-threads\fR arg (=4)
+Max number of threads to use when preparing block hashes in groups.
+.TP
+\fB\--fast-block-sync\fR arg (=1)
+Sync up most of the way by using embedded, known block hashes.
+.TP
+\fB\--block-sync-size\fR arg (=0)
+How many blocks to sync at once during chain synchronization (0 = adaptive).
+.TP
+\fB\--block-download-max-size\fR arg (=0)
+Set maximum size of block download queue in bytes (0 for default)
+.TP
+\fB\--no-sync\fR                   
```

Comment:
These options (especially the ones starting with extra-mesages-file) aren't really performance related.  I suggest folding them into the core server config section.